addons/project/tests/test_project_subtasks.py

```diff
@@ -356,3 +356,19 @@ def test_get_all_subtasks(self):
             self.assertEqual(parent_id, task.id, "The key should be the common ancestor")
             self.assertEqual(set(subtask_ids), set(all_subtasks.ids),
                              "All subtasks linked to the common ancestor should be returned by _get_subtask_ids_per_task_id method")
+
+    def test_subtask_is_visible_after_archiving(self):
+        """
+            Check if `display_in_project` of subtask is set to `True` once the subtask is archived
+        """
+        subtask = self.env['project.task'].create([
+            {
+                'name': 'Subtask',
+                'parent_id': self.task_1.id,
+                'project_id': self.project_pigs.id,
+                'display_in_project': False,
+            },
+        ])
```

Comment:
no need to use the a list, a dict is enough.
```py
 subtask = self.env['project.task'].create({
                'name': 'Subtask',
                'parent_id': self.task_1.id,
                'project_id': self.project_pigs.id,
                'display_in_project': False,
})
```

Reply:
Indeed, it's fixed now :)